test/org/traccar/protocol/Tk103ProtocolEncoderTest.java

```diff
@@ -31,6 +91,15 @@ public void testEncodePositionSingle() throws Exception {
 
         Assert.assertEquals("(123456789012345AP00)", encoder.encodeCommand(command));
 
+        IdentityManager old = Context.getIdentityManager();
+        Context.init(t580wManager);
+
+        try {
+            Assert.assertEquals("[begin]sms2,*getposl*,[end]", encoder.encodeCommand(command));
+        } finally {
```

Comment:
This is just wrong.

Reply:
as you suggested above, i will implement exactly the same approach as it was done in GT06.